tests/containers/netavark_integration.pm

```diff
@@ -32,6 +33,12 @@ sub run_tests {
     my $ret = script_run "env $env PATH=/usr/local/bin:\$PATH bats --tap test | tee -a $log_file", 1200;
 
     my @skip_tests = split(/\s+/, get_var('NETAVARK_BATS_SKIP', ''));
+
+    # Unconditionally ignore these flaky subtests
+    my @must_skip = ();
+    push @must_skip, "100-bridge-iptables" if ($firewalld_backend ne "iptables");
```

Comment:
shouldn't the firewall backend detection be above this line?

Reply:
We do it in the main run function.

Reply:
right ... I have misread the function name ... `run` vs `run_test`